stratified-split/script.whizzml

```diff
@@ -0,0 +1,82 @@
+;; Transposes the matrix: [[a b] [c d]] to [[a c] [b d]]
+
+(define (_transpose m)
+  (apply map list m))
+
+;; Creates the each dataset by sampling and using out-of-bag
+
+(define (sample-dataset ds-id rate oob)
+  (create-and-wait-dataset {"sample_rate" rate
+                            "origin_dataset"  ds-id
+                            "out_of_bag" oob
+                            "seed" "bigml"}))
+
+;; Splits the original dataset in two complementary datasets using
+;; a given sample rate
+
+(define (split-dataset ds-id rate)
```

Comment:
we have `create-dataset-split` in the standard library, and you can pass to it additional params (useful for setting the names of the two halves).  it also works in parallel.